components/ntp_background_images/browser/ntp_sponsored_images_data.cc

```diff
@@ -145,25 +148,70 @@ NTPSponsoredImagesData::NTPSponsoredImagesData(
     return;
   }
 
+  url_prefix = base::StringPrintf("%s://%s/", content::kChromeUIScheme,
+                                  kBrandedWallpaperHost);
   if (auto* name = json_value->FindStringKey(kThemeNameKey)) {
     theme_name = *name;
-    DVLOG(2) << __func__ << ": Theme name: " << theme_name;
+    url_prefix += kSuperReferralPath;
+  } else {
+    url_prefix += kSponsoredImagesPath;
+  }
+
+  auto* campaigns_value = json_value->FindListKey(kCampaignsKey);
+  if (campaigns_value) {
+    ParseCampaignsList(campaigns_value->Clone(), installed_dir);
+  } else {
+    // Get global campaign directly if campaigns list isn't existed.
```

Comment:
Typo?

Reply:
Updated to `Get a global campaign directly if the campaign list doesn't exist.`.